frontend/templates/views/partials/component-intro.j2

```diff
@@ -1,5 +1,11 @@
 {% do doc.styles.addCssFile('css/components/organisms/component-intro.css') %}
 <div class="ap-o-component-intro">
+  {% if doc.bentoPath and not doc.bento %}
+  {% do doc.icons.useIcon('icons/speech-bubble.svg') %}
+{% do doc.styles.addCssFile('css/components/molecules/tip.css') %}
+  <div class="ap-m-tip"> <div class="ap-a-ico ap-m-tip-icon"> <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#speech-bubble"></use></svg> </div> <div class="ap-m-tip-content">You can now use this component outside valid AMP documents using the <a href="{{ doc.bentoPath }}">experimental Bento version of this component</a>. Learn more in our <a href="../../guides-and-tutorials/start/bento_guide.md">Bento guide</a>.</div>
```

Comment:
Depends on #5198 - otherwise this will cause a broken link which will most probably break build:

![Bildschirmfoto 2021-01-26 um 16 37 41](https://user-images.githubusercontent.com/12857772/105867040-eabf5a00-5ff4-11eb-8d05-d5c8a1580a7d.png)

Reply:
Good point. Removed for now.